packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -175,7 +175,7 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> implements IP
                 ) }
               style={ customWidthStyles }
               elementToFocusOnDismiss={ elementToFocusOnDismiss }
-              isClickableOutsideFocusTrap={ true }
+              isClickableOutsideFocusTrap={ focusTrapZoneProps && !focusTrapZoneProps.isClickableOutsideFocusTrap ? false : true }
```

Comment:
Is there a reason to remove isLightDismiss || isHiddenOnDismiss ?

Reply:
superfluous now that we are switching the default to 'true'. If the props isLightDismiss or isHiddenOnDismiss were set to 'true' before it would set this to 'true' so clicks wouldn't be blocked.

Reply:
makes sense.